docs/run-node/run-node.md

```diff
@@ -38,6 +38,7 @@ The `~/.simapp` folder has the following structure:
       |- node_key.json              # Private key to use for node authentication in the p2p protocol.
       |- priv_validator_key.json    # Private key to use as a validator in the consensus protocol.
 ```
+If you want change any field values in the json files(ex: genesis.json) you can use `jq` ([installation](https://stedolan.github.io/jq/download/) & [docs](https://stedolan.github.io/jq/manual/#Assignment)) command to do that (for ex: ```jq '.chain_id = "testing1"' genesis.json > temp.json && mv temp.json genesis.json```)
```

Comment:
```suggestion
If you want change any field values in the json files (for ex: genesis.json) you can use `jq` ([installation](https://stedolan.github.io/jq/download/) & [docs](https://stedolan.github.io/jq/manual/#Assignment)) command to do that. For example, to change the genesis chain ID:
```bash
jq '.chain_id = "testing1"' genesis.json > temp.json && mv temp.json genesis.json
```
```

Reply:
I believe this should be a new section for better visibility. Eg `## Updating genesis fields`